### DISCUSSION TITLE
Have error with contianer mastodon_web_1

### DISCUSSION THREAD

**Discussion by raymand211092** [mastodon/mastodon#20165] [Q&A]
=> Run `bin/rails server --help` for more startup options
Exiting
/opt/ruby/lib/ruby/3.0.0/fileutils.rb:253:in `mkdir': Permission denied @ dir_s_mkdir - /opt/mastodon/tmp/pids (Errno::EACCES)
        from /opt/ruby/lib/ruby/3.0.0/fileutils.rb:253:in `fu_mkdir'
        from /opt/ruby/lib/ruby/3.0.0/fileutils.rb:231:in `block (2 levels) in mkdir_p'
        from /opt/ruby/lib/ruby/3.0.0/fileutils.rb:229:in `reverse_each'
        from /opt/ruby/lib/ruby/3.0.0/fileutils.rb:229:in `block in mkdir_p'

**Comment by MrGeneration** [comment]
You did `chown` the `public/` directory of the compose to user and group id `991`, did you? :)
